app/src/main/java/com/door43/translationstudio/core/TargetTranslationMigrator.java

```diff
@@ -133,6 +140,30 @@ private static File v5(File path) throws Exception {
     }
 
     /**
+     * make sure the license file is present in folder
+     * @param targetTranslationDir
+     */
+    private static void addLicenseFile(File targetTranslationDir) {
+        //ensure that there is a license file
+        try {
+            File license = new File(targetTranslationDir, LICENSE);
```

Comment:
please don't use static variables in the migrator. We cannot depend on the consistency of paths, or object properties between different version of the app. Therefore everything should be hard coded to avoid regression.